### ISSUE TITLE
The Tauren Chieftains do not seem to play in the World's End Tavern.

### ISSUE THREAD

**Issue by heyitsbench** [azerothcore/azerothcore-wotlk#14912]
### Current Behaviour

There is no band playing in the World's End Tavern.

### Expected Blizzlike Behaviour

The Tauren Chieftains band should be playing in the World's End Tavern in Shattrath every so often, complete with music and effects. Shattrath Saul should also introduce the band ten-ish minutes before they play, walking around the lower city all the while.

### Source

Wrath Classic

### Steps to reproduce the problem

1. `.tele shattrath`
2. Go to the tavern.
3. Wait.
4. Observe no band.

### Extra Notes

The timing mentioned on multiple websites is every three hours every day, but this is not entirely true. I'm not sure what the timing actually is, it does seem to be every three hours with some exceptions. In my quick look on Classic, they played at six P.M. server/local time, but not at nine P.M. I don't know why, but there are definitely some exceptions or nuance to the timings.

I did sniff this, recording can be found [here](https://www.youtube.com/watch?v=G1VmGCCOhGY).

### AC rev. hash/commit

e7e119283f776746f34f409480c567e87f078483

### Operating system

Windows 10 x64

### Custom changes or Modules

[mod-ptr-template](https://github.com/heyitsbench/mod-ptr-template)

**Comment by heyitsbench** [comment]
All times noted are both local/server time (Eastern Standard Time).

Spotted on Wrath Classic at:
6 PM, Wednesday (2/8/23)
2 PM, Thursday (2/9/23)

**Comment by heyitsbench** [comment]
I did double check just in case, this performance has no impact on the band placed in Silvermoon City, so there can be two copies of the band in two separate places.

**Comment by heyitsbench** [comment]
It seems to me that the concerts played by L70ETC are all the same regardless of location. The BlizzCon concert seems to only be triggered hourly, the one in Shattrath's tavern happens every four hours (from 2AM), and the one in BRD triggers daily at 8PM and can be manually triggered by a character with a Direbrew remote. I have sniffed the latter two (obviously no way I can sniff the former), and I have listed out what I've observed the creatures doing during the performance [here](https://gist.github.com/heyitsbench/47df255b86d580c4b3def178437f1ae1). Some creatures (the controllers and the goblin creatures/drumset in BRD) are permanently spawned, the band members themselves are only spawned when the show begins.
